### PR TITLE
Fix a few typos

### PR DESCRIPTION
**Pull request by kfogel**
### brief description of changes

These are simple typo fixes -- the diff pretty much says it all.

--

This box **must** be checked
- [x] the PR branch was originally made from the base branch

This box **should** be checked
- [ ] after these changes the docs build locally without error

(I confess I did not actually test the build after making these changes.  If these tiny edits break the build, I would be astonished!)

This box **should only** be checked you intend to follow through on it (we can do it on our end too)
- [ ] I will `cherry-pick` all commits in this PR into other branches that should have them _after_ this PR is merged

### CODE REVIEW

**Comment by kfogel** [comment]
@ekansa Easy PR here, if you're looking for an appetizer :-).

File: docs/administering/designing-the-database.rst [hunk 1]
```diff
@@ -63,7 +63,7 @@ Along the way, you can use the preview button to display the graph in a more gra
 Core Arches Datatypes
 ---------------------
 
-Nodes in Arches must be configured with a "Data Type", and different datatypes store different kinds of information. For example, a **string** datatype is what you should use to store arbitrary text, like the name or description of a resource. A brief description of all datatype options in core Arches follows. Developers and extend Arches by :ref:`creating their own custom datatype <datatypes>`.
+Nodes in Arches must be configured with a "Data Type", and different datatypes store different kinds of information. For example, a **string** datatype is what you should use to store arbitrary text, like the name or description of a resource. A brief description of all datatype options in core Arches follows. Developers can extend Arches by :ref:`creating their own custom datatypes <datatypes>`.
```
Comment (ekansa) [inline docs/administering/designing-the-database.rst:66]: Nice catch @kfogel ! Thanks!

File: docs/developing/reference/import-export.rst [hunk 2]
```diff
@@ -1093,7 +1093,7 @@ Below is an example of a simple resource model view. If a resource instance has
             AND record.tiledata->>'677f2c0f-09cc-11e7-b412-6c4008b05c4c'
             != ''
 
-#. We can then define a field be referencing that tile:
+#. We can then define a field by referencing that tile:
```
Comment (ekansa) [inline docs/developing/reference/import-export.rst:1096]: One letter change improves clarity immensely!  Thanks @kfogel !

**Review by ekansa — APPROVED** [review]

File: docs/developing/reference/import-export.rst (hunk 2 above)
Comment (kfogel) [reply docs/developing/reference/import-export.rst:1096]: Thanks for the review & merge, @ekansa.  I realize they're small contributions, but I figure as long as I'm spotting minor glitches here and there while reading, I might as well send in patches for them.  (I'll always try to keep typo stuffseparate from any substantive doc contributions.)

**Pull request merged**